models/ecoli/analysis/multigen/sub_gen_gene_expr_NV.py

```diff
@@ -0,0 +1,80 @@
+"""
+Plot to identify candidate sub generational genes
+"""
+from __future__ import absolute_import, division, print_function
+
+import pickle
+import os
+
+from wholecell.utils.sparkline import whitePadSparklineAxis
+import numpy as np
+from matplotlib import pyplot as plt
+from models.ecoli.analysis import multigenAnalysisPlot
+from wholecell.analysis.analysis_tools import (exportFigure, read_stacked_columns)
+from wholecell.io.tablereader import TableReader
+
+
+class Plot(multigenAnalysisPlot.MultigenAnalysisPlot):
+	def do_plot(self, seedOutDir, plotOutDir, plotOutFileName, simDataFile, validationDataFile, metadata):
+
+		with open(simDataFile, 'rb') as f:
+			sim_data = pickle.load(f)
+
+		cell_paths = self.ap.get_cells()
+
+		simOutDir = os.path.join(cell_paths[0], "simOut")
+
+		time = read_stacked_columns(cell_paths, 'Main', 'time').squeeze()
+
+		cell_cycle_length = read_stacked_columns(cell_paths, 'Main', 'time',fun=lambda x: (x[-1] - x[0])).squeeze()
+
+		generation_time = np.cumsum(cell_cycle_length)
+
+		generation_index = np.searchsorted(time, generation_time)
+
+		# Determine all subgen genes
+		mRNA_counts_reader = TableReader(os.path.join(simOutDir, 'mRNACounts'))
+		mRNA_cistron_ids = mRNA_counts_reader.readAttribute('mRNA_cistron_ids')
+
+		all_cistron_counts = read_stacked_columns(
+			cell_paths, 'mRNACounts', 'mRNA_cistron_counts')
+
+		cistron_expressed_bool = (all_cistron_counts == 0)*1
```

Comment:
Should this be `(all_cistron_counts > 0)`?

Reply:
Oops